File: fix-cities.py
# Read the corrupted file
with open('src/data/cities.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace the corrupted patterns
content = content.replace('`r`n        status:', '\n        status:')

# Write back
with open('src/data/cities.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("File fixed!")

File: fix-cities.js
const fs = require('fs');

// Read the file
let content = fs.readFileSync('src/data/cities.ts', 'utf8');

// Replace the escaped characters
content = content.replace(/`r`n\s+status:/g, '\r\n        status:');

// Write back
fs.writeFileSync('src/data/cities.ts', content, 'utf8');

console.log('File fixed!');
